Chiazokam/iReporterApp
# 162551177 set up the database

#### What does this PR do?
- Sets up the database on ElephantSQL
- Refactors the createRecord controller to post data to the database

#### Description of Task to be completed?
- Set up the database
- Refactor the createRecord controller to post data to the database

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/iReporterApp`
- Enter into the project directory - `cd iReporterApp`
- switch to this branch - `checkout ch-setup-database-162551177`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Send a `POST` request to `http://localhost:3000/api/v1/redflags or http://localhost:3000/api/v1/interventions`
- You should receive a successful response

#### Any background context you want to provide?
- the POST endpoint payload should be structured like this- 
`
{
  "title": "story on the flute",
       "createdOn": "05-03-18",
       "createdBy": 1,
       "type": "intervention",
       "location": "3.74747, 4.76777",
       "comment": "Mama",
       "images": "bisi.gif",
       "videos": "girl.mp4"
     }
`

#### What are the relevant pivotal tracker stories?
[#162551177](https://www.pivotaltracker.com/story/show/162551177)
